sycl/plugins/level_zero/pi_level_zero.cpp

```diff
@@ -5157,27 +5164,27 @@ pi_result piEventGetProfilingInfo(pi_event Event, pi_profiling_info ParamName,
   switch (ParamName) {
   case PI_PROFILING_INFO_COMMAND_START: {
     ZE_CALL(zeEventQueryKernelTimestamp, (Event->ZeEvent, &tsResult));
-    uint64_t ContextStartTime = tsResult.context.kernelStart * ZeTimerFreq;
+    uint64_t ContextStartTime =
+        (tsResult.context.kernelStart & TimestampMaxValue) * ZeTimerResolution;
```

Comment:
Does it mean that the Level-Zero spec guidance needs to be adjusted?

https://spec.oneapi.io/level-zero/latest/core/PROG.html#kernel-timestamp-events